pkg/device/nvidia/device.go

```diff
@@ -99,27 +99,32 @@ const (
 )
 
 type NvidiaConfig struct {
-	ResourceCountName            string  `yaml:"resourceCountName"`
-	ResourceMemoryName           string  `yaml:"resourceMemoryName"`
-	ResourceCoreName             string  `yaml:"resourceCoreName"`
-	ResourceMemoryPercentageName string  `yaml:"resourceMemoryPercentageName"`
-	ResourcePriority             string  `yaml:"resourcePriorityName"`
-	OverwriteEnv                 bool    `yaml:"overwriteEnv"`
-	DefaultMemory                int32   `yaml:"defaultMemory"`
-	DefaultCores                 int32   `yaml:"defaultCores"`
-	DefaultGPUNum                int32   `yaml:"defaultGPUNum"`
-	DeviceSplitCount             uint    `yaml:"deviceSplitCount"`
-	DeviceMemoryScaling          float64 `yaml:"deviceMemoryScaling"`
-	DeviceCoreScaling            float64 `yaml:"deviceCoreScaling"`
+	NodeDefaultConfig            `yaml:",inline"`
```

Comment:
Please leave a comment here to inform that this config is shared by `Nodeconfig` struct